query-algebrizer/src/types.rs

```diff
@@ -274,20 +274,21 @@ impl From<Order> for OrderBy {
 }
 
 #[derive(Copy, Clone, PartialEq, Eq)]
-/// Define the different numeric inequality operators that we support.
+/// Define the different inequality operators that we support.
 /// Note that we deliberately don't just use "<=" and friends as strings:
 /// Datalog and SQL don't use the same operators (e.g., `<>` and `!=`).
-pub enum NumericComparison {
+/// These are applicable to numbers and instants.
+pub enum Inequality {
```

Comment:
Is `Inequality` entirely accurate when there are `LessThanOrEquals` and `GreaterThanOrEquals` as options on the enum?